js/renovation/ui/button.tsx

```diff
@@ -40,19 +41,18 @@ const getCssClasses = (model: ButtonProps) => {
 
   return classNames.join(' ');
 };
-
-export const viewFunction = (viewModel: Button) => {
+export const viewFunction = (viewModel: Button): JSX.Element => {
   const {
-    children, icon, iconPosition, template, text,
+    children, icon, iconPosition, template: ButtonTemplate, text,
   } = viewModel.props;
-  const renderText = !template && !children && text;
+  const renderText = !ButtonTemplate && !children && text;
```

Comment:
WTF? Why are tests passed?

Reply:
it is just a variable name

https://github.com/DevExpress/DevExtreme/pull/13873/files#diff-6bde74af2450bbb641a700bbb770adf2R46

Reply:
What is wrong?

Reply:
It is not a type. I think that ButtonTemplate is type of template.